package/python-kiwi-spec-template

```diff
@@ -501,6 +519,12 @@ fi
 %exclude %{_bindir}/kiwicompat-*
 %{_bindir}/*
 
+%files -n kiwi-dracut-iso
+%defattr(-, root, root)
+%dir %{_usr}/lib/dracut
+%dir %{_usr}/lib/dracut/modules.d
```

Comment:
Since you already require dracut, you should not attempt to co-own dracut directories.

Reply:
yes fixing